operator/apis/loki/v1/lokistack_webhook.go

```diff
@@ -16,33 +16,33 @@ import (
 
 // SetupWebhookWithManager registers the Lokistack to the controller-runtime manager
 // or returns an error.
-func (s *LokiStack) SetupWebhookWithManager(mgr ctrl.Manager) error {
+func (r *LokiStack) SetupWebhookWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewWebhookManagedBy(mgr).
-		For(s).
+		For(r).
 		Complete()
 }
 
-//+kubebuilder:webhook:path=/validate-loki-grafana-com-v1beta1-lokistack,mutating=false,failurePolicy=fail,sideEffects=None,groups=loki.grafana.com,resources=lokistacks,verbs=create;update,versions=v1beta1,name=vlokistack.kb.io,admissionReviewVersions=v1
+//+kubebuilder:webhook:path=/validate-loki-grafana-com-v1-lokistack,mutating=false,failurePolicy=fail,sideEffects=None,groups=loki.grafana.com,resources=lokistacks,verbs=create;update,versions=v1,name=vlokistack.kb.io,admissionReviewVersions=v1
```

Comment:
I noticed then forgot about this during a past review, but should we pick a better suffix for the validation webhooks? `.kb.io` seems like it is still coming from the sample code for kubebuilder.

Reply:
Yes that's a good idea to keep the same domain as for the CRD, i.e. renaming to `vlokistack.loki.grafana.com`